kibbeh/src/ui/GistPopup.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState, useEffect } from "react";
+import { Gist, File } from "../types/Gist";
+
+export interface GistPopupProps {
+  id: string;
+}
+
+export const GistPopup: React.FC<GistPopupProps> = ({ id }) => {
+  const [gistData, setGistData] = useState<Gist | null>(null);
+  const [selectedFile, setSelectedFile] = useState<File | null>(null);
+
+  useEffect(() => {
+    fetch(`https://api.github.com/gists/${id}`)
+      .then((res) => res.json())
+      .then(setGistData);
+  }, [id]);
+
+  if (!gistData) return <div>Loading...</div>;
+  const files = Object.values(gistData.files);
+  if (!selectedFile) {
+    setSelectedFile(files[0]);
+    return <div>Loading...</div>;
+  }
+
+  return (
+    <div className="w-full bg-primary-800 flex flex-col text-primary-100 max-w-2xl rounded-8 max-h-24">
```

Comment:
all blocks are flex by default, no need for `flex` classname
```suggestion
    <div className="w-full bg-primary-800 flex-col text-primary-100 max-w-2xl rounded-8 max-h-24">
```